Examples/Io/Performance/ActsExamples/Io/Performance/VertexPerformanceWriter.cpp

```diff
@@ -107,11 +107,19 @@ ActsExamples::VertexPerformanceWriter::VertexPerformanceWriter(
     m_outputTree->Branch("recoZ", &m_recoZ);
     m_outputTree->Branch("recoT", &m_recoT);
 
+    m_outputTree->Branch("seedX", &m_seedX);
+    m_outputTree->Branch("seedY", &m_seedY);
+    m_outputTree->Branch("seedZ", &m_seedZ);
+    m_outputTree->Branch("seedT", &m_seedT);
+
     m_outputTree->Branch("resX", &m_resX);
     m_outputTree->Branch("resY", &m_resY);
     m_outputTree->Branch("resZ", &m_resZ);
     m_outputTree->Branch("resT", &m_resT);
 
+    m_outputTree->Branch("resSeedZ", &m_resSeedZ);
```

Comment:
we skip X and Y because the seed is always 0 there right?

does this assumption hold for any kind of seeder?

Reply:
Yes!

It holds for any seeder that I know of.